oncokb_annotator_3.2.2/oncokb_annotator_3-2-2.cwl

```diff
@@ -0,0 +1,90 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: oncokb_annotator
+baseCommand:
+  - python3
+  - /oncokb/MafAnnotator.py
+inputs:
+  - id: inputMafFile
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '-i'
+    doc: input maf file for annotation
+  - id: outputMafName
+    type: string
+    inputBinding:
+      position: 0
+      prefix: '-o'
+  - id: apiToken
+    type: string
+    inputBinding:
+      position: 0
+      prefix: '-b'
+    doc: OncoKB API token
+  - id: previousResult
+    type: File?
+    inputBinding:
+      position: 0
+      prefix: '-p'
+  - id: clinicalFile
+    type: File?
+    inputBinding:
+      position: 0
+      prefix: '-c'
+    doc: |-
+      Essential clinical columns:
+          SAMPLE_ID: sample ID
+          ONCOTREE_CODE: tumor type code from oncotree (http://oncotree.mskcc.org)
+  - id: tumorType
+    type: string?
+    inputBinding:
+      position: 0
+      prefix: '-t'
+    doc: |-
+      Cancer type will be assigned based on the following priority:
+          1) ONCOTREE_CODE in clinical data file
+          2) ONCOTREE_CODE exist in MAF
+          3) default tumor type (-t)
+  - id: referenceGenome
```

Comment:
Can we also please add the option to annotate hotspot from the oncokb command line?